Car-Specs/Car-Specs/Models/Data/ModelsModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Car_Specs.Models.Data
+{
+    public class ModelsModel
+    {
+
+        [BsonElement("modelId")]
```

Comment:
There is a way to configure the property name convention while setting up mongoClient. You can check that. 
check here [https://jonathancrozier.com/blog/interacting-with-mongodb-using-c-sharp](https://jonathancrozier.com/blog/interacting-with-mongodb-using-c-sharp)